PlayerConfigurationRegister/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/playerconfigurationregister/sendingplayers/PlayerConnectedEvent.java

```diff
@@ -0,0 +1,34 @@
+package pl.jakubowskiprzemyslaw.tajgertim.playerconfigurationregister.sendingplayers;
+
+import org.springframework.context.ApplicationEvent;
+import pl.jakubowskiprzemyslaw.tajgertim.models.configuration.PlayerConfiguration;
+
+import java.util.Objects;
+
+class PlayerConnectedEvent extends ApplicationEvent {
+
+    private static final long serialVersionUID = 570102554291869796L;
+    private final PlayerConfiguration playerConfiguration;
+
+    PlayerConnectedEvent(Object source, PlayerConfiguration playerConfiguration) {
+        super(source);
+        this.playerConfiguration = playerConfiguration;
+    }
+
+    PlayerConfiguration getPlayerConfiguration() {
+        return playerConfiguration;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```

Comment:
It could be replaced by instanceOf

Reply:
as said during review "you can do it, but don't have to"